### PR TITLE
Address IANA comment on Section 5.2 (now Section 6.2).

### PR DESCRIPTION
**Pull request by cabo**
This is a minimal change to address the observation in the IANA comments.

### CODE REVIEW

File: draft-ietf-core-problem-details.md [hunk 1]
```diff
@@ -588,7 +589,14 @@ Initial entries in this sub-registry are as follows:
 
 This specification defines a new sub-registry for Custom Problem
 Detail Keys in the CoRE Parameters registry {{!IANA.core-parameters}},
-with the policy "first come first served" {{!RFC8126}}.
+with the policy "expert review" ({{Section 4.5 of -ianacons}}).
+
+The expert is instructed to attempt making the registration experience
+as close to first-come-first-served as reasonable achievable, but
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]:
```suggestion
as close to first-come-first-served as reasonably achievable, but
```

File: draft-ietf-core-problem-details.md [hunk 2]
```diff
@@ -588,7 +589,14 @@ Initial entries in this sub-registry are as follows:
...
+checking that the reference document does provide a description as set
+out below.
+(This requirement is a relaxed version of "specification required" as
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md:598]: Not sure this is necessary

**Review by thomas-fossati — COMMENTED** [review]
couple of small comments

File: draft-ietf-core-problem-details.md (hunk 2 above)
Comment (cabo) [reply draft-ietf-core-problem-details.md:598]: I wrote this mainly to fend of the knee-jerk "but this is like specification required, why don't you use that?".

Comment (thomas-fossati) [reply draft-ietf-core-problem-details.md:598]: mmm, reminds me of St Jerome's letter 4: _"[...] dum excusare credis, accusas"_, but OK :-)

**Review by thomas-fossati — APPROVED** [review]

**Pull request merged**